action.yml

```diff
@@ -15,7 +15,7 @@ inputs:
   number-of-days: # optional, defaults to 30
     description: number of days to scan
     required: true
-    default: 30
+    default: "30"
```

Comment:
this is an integer - so shouldn't have "" around it (everything else is a string).

Reply:
Was thinking the same until my linter told me otherwise.
![image](https://user-images.githubusercontent.com/2204092/190423670-f7fb3d6c-3826-401e-8989-54923d2ccf92.png)